.circleci/config.yml

```diff
@@ -20,17 +20,9 @@ workflows:
       - shellcheck/check:
           exclude: SC2148,SC2038,SC2086,SC2002,SC2016,SC1090
```

Comment:
Do we still need these exclusions? Let's try removing them and seeing if they are necessary.

Reply:
Good call, I was able to fix and remove 4 of them.